src/metrickit.cs

```diff
@@ -141,20 +141,20 @@ interface MXMetaData : NSSecureCoding {
 		[Export ("JSONRepresentation")]
 		NSData JsonRepresentation { get; }
 
-#if !MONOMAC
+		[NoMac]
 		[Internal]
 		[Deprecated (PlatformName.iOS, 14,0)]
 		[Export ("DictionaryRepresentation")]
 		NSDictionary _DictionaryRepresentation13 { get; }
 
-		[Internal]
-		[iOS (14,0)]
-		[MacCatalyst (14,0)]
-		[Export ("dictionaryRepresentation")]
-		NSDictionary _DictionaryRepresentation14 { get; }
-#else 
-		[Export ("dictionaryRepresentation")]
-		NSDictionary DictionaryRepresentation { get; }
+ 		[iOS (14,0)]
```

Comment:
Looks like there's a space at the beginning of each line now?

Reply:
Strange, not sure how that got there...fixing.